src/webextension/list/manage/components/account-summary.js

```diff
@@ -18,8 +18,8 @@ export function AccountSummaryButton({displayName, avatar, onClick}) {
 
   return (
     <button className={styles.button} onClick={onClick}>
-      <span className={styles.avatar}><img src={avatar}/></span>
       <span className={styles.displayName}>{displayName}</span>
+      <span className={styles.avatar}><img src={avatar}/></span>
```

Comment:
Moving the avatar from first to last requires some changes to the unit tests to account for the change in position:

*  [account-summary-test.js line 26](https://github.com/mozilla-lockbox/lockbox-extension/blob/c87cb321abc383e295f491a1b261488555ad2423/test/unit/list/manage/components/account-summary-test.js#L26]) -- `expect(wrapper.find("span").at(1))` should be `expect(wrapper.find("span").at(0))`
* [account-summary-test.js line 115](https://github.com/mozilla-lockbox/lockbox-extension/blob/c87cb321abc383e295f491a1b261488555ad2423/test/unit/list/manage/components/account-summary-test.js#L115) -- `expect(wrapper.find("span").at(1))` should be `expect(wrapper.find("span").at(0))`
* [current-account-summary-test.js line 47](https://github.com/mozilla-lockbox/lockbox-extension/blob/c87cb321abc383e295f491a1b261488555ad2423/test/unit/list/manage/containers/current-account-summary-test.js#L47) -- `expect(wrapper.find("span").at(1))` should be `expect(wrapper.find("span").at(0))`